js/filters.js

```diff
@@ -115,9 +145,27 @@ function TypeaheadFilter(elm) {
   var key = this.$body.data('dataset');
   var dataset = typeahead.datasets[key];
   this.typeaheadFilter = new typeaheadFilter.TypeaheadFilter(this.$body, dataset);
```

Comment:
Ah okay, that makes sense, thanks for the clarification!  +1 on the separate patch for any work involved in merging the two.